include/xten/Transform/Passes.td

```diff
@@ -35,8 +35,19 @@ def LowerToLibATen : Pass<"lower-to-libaten", "ModuleOp"> {
 }
 
 def XTenMinimizeLiveTensors : Pass<"xten-minimize-live", "func::FuncOp"> {
-  let summary = "reorder xten operations to minimize the total size of the live tensors";
+  let summary = "Reorder xten operations to minimize the total size of the live tensors";
   let constructor = "xilinx::xten::createXTenMinimizeLiveTensorsPass()";
+  let options = [
+    Option<"printSchedCost", "print-sched-cost", "bool", /*default=*/"false",
+           "Prints a dot representation of the cost associated with operations "
+           "of the function in stderr.">,
+    Option<"printBeforeSched", "print-before-sched", "bool", /*default=*/"false",
+           "Prints a dot representation of the sequence of operations in the DAG "
+           "in the order they are to be executed before scheduling in stderr.">,
+    Option<"printAfterSched", "print-after-sched", "bool", /*default=*/"false",
+           "Prints a dot representation of the sequence of operations in the DAG "
+           "in the order they are to be executed after scheduling is done in stderr.">,
```

Comment:
Please add some minimal level of testing, that includes the main features of each print. 

I also wonder if `print` is the common term used here. I think that us usually used for a human-readable output whereas `--dot` is used in https://llvm.org/docs/Passes.html. In MLIR I only see https://mlir.llvm.org/docs/Passes/#-view-op-graph-print-graphviz-visualization-of-an-operation.